Makefile

```diff
@@ -304,7 +304,17 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool $(CONTAINER_TOOL) --mode semver --tag $(CATALOG_IMG) -i quay.io/operator-framework/opm:$(OPM_VERSION)-ppc64le --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	$(shell mkdir rsct-temp)
+	$(eval TMP_DIR := rsct-temp)
+	@envsubst < catalog/preamble_config_template.json > $(TMP_DIR)/rsct-operator-catalog.json
+	$(OPM) render $(BUNDLE_IMGS) >> $(TMP_DIR)/rsct-operator-catalog.json
+	$(OPM) generate dockerfile $(TMP_DIR) -i quay.io/operator-framework/opm:${OPM_VERSION}
+    ## Apending Final stage for ppc64le
+	echo "FROM quay.io/operator-framework/opm:$(OPM_VERSION)-ppc64le" >> $(TMP_DIR).Dockerfile
```

Comment:
you can include this line into the below `Dockerfile_final_stage` docker file itself and use ARG for replacing VERSION substitution

Reply:
I tried that approach, but it doesn't work because the ARG gets declared in the first stage and isn't accessible in the second stage.

Reply:
can you adopt this `--platform=linux/ppc64le` instead of `-ppc64le`? long term solution - lets see if we can work with OPM team to generate the Dockerfile itself to support any target architecture(enhance generate command)

Reply:
I have tried using `--platform=linux/ppc64le`, it works locally but in workflow it pull x86 image.

Reply:
> long term solution - lets see if we can work with OPM team to generate the Dockerfile itself to support any target architecture(enhance generate command)

sure